website/content/commands/pipeline-run.mdx

```diff
@@ -0,0 +1,44 @@
+---
+layout: commands
+page_title: 'Commands: Pipeline run'
+sidebar_title: 'pipeline run'
+description: 'Manually execute a pipeline by name.'
+---
+
+# Waypoint Pipeline run
+
+Command: `waypoint pipeline run`
+
+Manually execute a pipeline by name.
+
+@include "commands/pipeline-run_desc.mdx"
+
+## Usage
+
+Usage: `waypoint pipeline run [options] <pipeline-name>`
+
+Run a pipeline by name. If run outside of a project dir, a '-project' flag is
+required.
+
+#### Global Options
+
+- `-plain` - Plain output: no colors, no animation. The default is false.
+- `-app=<string>` (`-a`) - App to target. Certain commands require a single app target for Waypoint configurations with multiple apps. If you have a single app, then this can be ignored.
+- `-project=<string>` (`-p`) - Project to target.
+- `-workspace=<string>` (`-w`) - Workspace to operate in.
+
+#### Operation Options
+
+- `-label=<key=value>` - Labels to set for this operation. Can be specified multiple times.
+- `-local` - True to use a local runner to execute the operation, false to use a remote runner.
+  If unset, Waypoint will automatically determine where the operation will occur,
+  defaulting to remote if possible.
+- `-remote-source=<key=value>` - Override configurations for how remote runners source data. This is specified to the data source type being used in your configuration. This is used for example to set a specific Git ref to run against.
+- `-var=<key=value>` - Variable value to set for this operation. Can be specified multiple times.
+- `-var-file=<string>` - HCL or JSON file containing variable values to set for this operation. If any "_.auto.wpvars" or "_.auto.wpvars.json" files are present, they will be automatically loaded.
+
+#### Command Options
+
+- `-pipeline-id=<string>` - Run a pipeline by ID.
```

Comment:
The usage block reads "run a pipeline by name" are ID and Name the same here?

Reply:
Name refers to the name from `pipeline "name" {}` where id refers to the uuid from its entry in the database, so they are different in this case. I think the default/expected case here is running by name and ID might be a special case, so I made ID a flag and name an argument.